CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 ### Added
 * `asf_tools.flood_map` and an associated `flood_map` entrypoint for making
   flood depth maps with products generated by `asf_tools.water_map`.
+* Notes to `README` and `asf_tools.flood_map.make_flood_map` stating that
+  this functionality is still under active development.
```

Comment:
I slightly prefer this as a single changelog entry; it feels odd to document changes to something we only added in this release. Something like:

> `asf_tools.flood_map` and an associated `flood_map` entrypoint for making
  flood depth maps with products generated by `asf_tools.water_map`. This functionality is still under active development and the products created using this function are likely to change in the future.